packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -19,6 +19,84 @@ const RANGE_KEY_TYPE = 'RANGE';
 // https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Limits.html#limits-secondary-indexes
 const MAX_LOCAL_SECONDARY_INDEX_COUNT = 5;
 
+/**
+ * Options for configuring a request latency metric that considers multiple operations.
+ */
+export interface SuccessfulRequestLatencyForOperationsMetricOptions extends cloudwatch.MetricOptions {
+
+  /**
+   * The operations to apply the metric to.
+   */
+  readonly operations: Operation[];
+
+}
+
+/**
+ * Options for configuring a system errors metric that considers multiple operations.
+ */
+export interface SystemErrorsForOperationsMetricOptions extends cloudwatch.MetricOptions {
+
+  /**
+   * The operations to apply the metric to.
+   *
+   * @default - All operations available by DynamoDB tables will be considered.
+   */
+  readonly operations?: Operation[];
+
+}
+
+/**
+ * Supported DynamoDB table operations.
+ */
+export enum Operation {
+
+  /**
+   * GetItem.
+   */
+  GET_ITEM = 'GetItem',
+
+  /**
+   * BatchGetItem.
+   */
+  BATCH_GET_ITEM = 'BatchGetItem',
```

Comment:
`~` (Super-nitpicky) Totally stylistic, but for these kinds of enums with redundant docstrings, I'd prefer to inline to reduce wasted space.

```suggestion
  /** GetItem */
  GET_ITEM = 'GetItem',
  /** BatchGetItem */
  BATCH_GET_ITEM = 'BatchGetItem',
```